indico/modules/users/forms.py

```diff
@@ -70,6 +70,16 @@ class UserPreferencesForm(IndicoForm):
         widget=SwitchWidget(),
         description=_('The previewer is used by default for image and text files, but not for PDF files.'))
 
+    add_ical_alerts = BooleanField(
+        _('Add alerts to iCal'),
+        widget=SwitchWidget(),
+        description=_('Add an event reminder to exported iCal files/URLs.'))
+
+    add_ical_alerts_mins = IntegerField(
+        _('Calendar notification time in minutes'),
```

Comment:
```suggestion
        _('iCal notification time'),
```

That's a bit shorter (maybe even short enough to not trigger wrapping, but often hard to avoid..), and the description is a better place to put anything lengthy